### DISCUSSION TITLE
求助！请问原神的新深渊该怎么打？

### DISCUSSION THREAD

**Discussion by McQueen979** [share121/inter-knot#26] [其他]
![02爱莉希雅 真我](https://github.com/user-attachments/assets/73585e50-63fa-4ec4-8283-e833d201228d)

感觉一下子要养成好多角色，还有驱动盘、声骸、光锥、圣痕都要升级，好多限制呀

**Comment by Zzz-walnut** [comment]
这简单啊，首先速度鞋是必备的，保证队伍多动，其次要带击破位快速累积失衡值，冰队狼哥电队安比，当然可以等一手青衣，重中之重的是，充能效率一定要够，先保证队伍循环再考虑双暴攻击，如果紊乱队记得堆精通

**Reply by McQueen979** [reply]
冰队可以用申鹤拐吗？我也有镜流专武能不能代替

**Comment by YumeRin39Hana** [comment]
原神是什么游戏吗？我不是很清楚，但你提到了驱动盘，我有货......感兴趣私聊.......

**Reply by McQueen979** [reply]
原神你没玩过吗？手机拿来我给你下载。驱动盘dd

**Comment by McQueen979** [comment]
驱动盘DD

**Comment by YumeRin39Hana** [comment]
有意加+13653811
